File: 백준/Bronze/14681. 사분면 고르기/사분면 고르기.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
let input = fs.readFileSync(0).toString().split("\n");

solution(parseInt(input[0]), parseInt(input[1]));

function solution(A, B) {
  if (A > 0 && B > 0) {
    console.log(1);
  } else if (A < 0 && B > 0) {
    console.log(2);
  } else if (A < 0 && B < 0) {
    console.log(3);
  } else if (A > 0 && B < 0) {
    console.log(4);
  }
}

File: 백준/Bronze/14681. 사분면 고르기/사분면 고르기.py
# 1쿼터 양수,양수 2쿼터 음수,양수 3쿼터 음수,음수 4쿼터 양수,음수

A,B = int(input()),int(input())

if A > 0 and B > 0 :
    print('1')
elif A < 0 and B > 0 :
    print('2')
elif A < 0 and B < 0 :
    print('3')
elif A > 0 and B < 0 :
    print('4')